lib/classes/compatibility/buddyboss.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Plugin Name: BuddyPress
+ * Plugin URI: https://www.buddyboss.com/platform/
+ *
+ * Compatibility Description: Ensures compatibility with BuddyBoss.
+ *
+ */
+
+namespace wpCloud\StatelessMedia {
+
+    if(!class_exists('wpCloud\StatelessMedia\BuddyBoss')) {
+        
+        class BuddyBoss extends ICompatibility {
+            protected $id = 'buddyboss';
+            protected $title = 'BuddyBoss';
+            protected $constant = 'WP_STATELESS_COMPATIBILITY_BUDDYPRESS';
```

Comment:
buddypress or buddyboss?

Reply:
It should have been `BuddyBoss`. Updated the text.